pioneer/packages/joy-utils/src/transport/proposals.ts

```diff
@@ -177,7 +210,7 @@ export default class ProposalsTransport extends BaseTransport {
   }
 
   async subscribeProposal (id: number|ProposalId, callback: () => void) {
-    return this.proposalsEngine.proposals(id, callback);
+    return this.api.query.proposalsEngine.proposals(id, callback);
```

Comment:
this one here I understand because you want to subsribe to updates..?

Reply:
Yes, this is also not available via cached api. Only simple queries are supported.

Reply:
ok thanks for clarification